src/RequestBuilder.php

```diff
@@ -107,6 +108,12 @@ public function build($path, Message $message, array $headers = [])
                 // We found a valid uriTemplate - now build and return the Request
 
                 list($body, $queryParams) = $this->constructBodyAndQueryParameters($message, $config);
+                
+                // Request enum fields be encoded as numbers rather than strings.
```

Comment:
nit: I'm reading this sentence in a caveman voice

Reply:
haha oops, fixed.